forge/test/mlir/test_ops.py

```diff
@@ -87,18 +90,23 @@ def __init__(self, dims):
         def forward(self, a):
             return torch.transpose(a, *self.dims)
 
+    # currently the lowering to TTNN is not supported for this case
+    compiler_cfg = forge.config._get_global_compiler_config()
+    compiler_cfg.compile_depth = forge.CompileDepth.FINISH_COMPILE
```

Comment:
This way, you'll limit all Transpose tests to run until the specified compile depth. It'll be good to early stop only those which are problematic, and run e2e the ones that are functional. 

To do so, you can detect dim args from test param, and lower compile depth only in those cases :))

Reply:
I was thinking about that, will do that :)